docs/the_bevy_organization.md

```diff
@@ -0,0 +1,78 @@
+
+We choose new SMEs carefully and only after they have proven themselves in the Bevy community. SMEs must have a proven track record of the following:
+
+1. **Designing and contributing to foundational pieces in their subject area**: SMEs are responsible for building and extending the foundations of a given subject area. They must have a history of doing this before becoming an SME.
+2. **Thorough reviews of other peoples' PRs in their subject area**: Within a subject area, SMEs are responsible for guiding people in the correct technical direction and only approving things aligned with that vision. They must have a history of doing this before becoming an SME.
+3. **Great social skills**: Within a subject area, SMEs are responsible for reviewing peoples' code, communicating project vision, and establishing consensus. They are representatives of the project and their actions directly reflect our goals and values. Working with them should not be painful.
+
+To make it easy to reach consensus, hold a high quality bar, and synchronize vision, we intentionally keep the number of SMEs in a given area small: 2 is the absolute minimum (to allow voting to occur), 3 is preferred, and 4 will be allowed in some cases. Bevy Organization members can be SMEs in more than one area, and Maintainers can also be SMEs.
+
+If you are interested in a SME role and believe you meet these criteria, reach out to one of our Project Leads or Maintainers. One month after every Bevy release Maintainers and Project Leads will evaluate the need for new roles, review candidates, and vote. Bringing in a new SME requires the support of the Project Leads and half of the Maintainers (however unanimous support is preferred).
+
+Our current Subject Matter Experts:
+
+(Finalizing this list now)
+
+### Bevy Org Member / Triage Team
+
+[Bevy Org members](https://github.com/orgs/bevyengine/people) are contributors who:
+
+1. Have actively engaged with Bevy development.
+2. Have demonstrated themselves to be polite and welcoming representatives of the project with an understanding of our goals and direction.
+3. Have asked to join the Bevy Org. Reach out to @cart on [Discord] or email us at bevyengine@gmail.com if you are interested. Everyone is welcome to do this. We generally accept membership requests, so don't hesitate if you are interested!
+
+All Bevy Org members are also Triage Team members. The Triage Team can label and close issues and PRs but do not have merge rights or any special authority within the community.
```

Comment:
This could also mention that org members are allowed to do `bors try`

Reply:
Good call!

Reply:
```suggestion
All Bevy Org members are also Triage Team members. The Triage Team can label and close issues and PRs but do not have merge rights or any special authority within the community.
Org members also have the ability to use `bors try`, causing our build system to evaluate the PR again.
```

Reply:
Bors also run additional jobs that aren't run by normal ci: iOS, Android, windows, no default features

Reply:
Updated, thanks.